tests/integration/models/application.spec.coffee

```diff
@@ -191,36 +191,30 @@ describe 'Application Model', ->
 				promise = resin.models.application.getApiKey(999999)
 				m.chai.expect(promise).to.be.rejectedWith('Application not found: 999999')
 
+	describe 'when toggling device URLs', ->
+		beforeEach ->
+			resin.models.application.create('DeviceUrlsTestApp', 'raspberry-pi').then (application) =>
+				@application2 = application
```

Comment:
Looking closer, I _think_ you can drop the `application2` part of this. You can just use the existing `@application` - it doesn't matter if you mess with it, because we delete all apps and devices and recreate everything between every single test.